config/locales/en.yml

```diff
@@ -202,7 +202,7 @@ en:
         version_name: For datasets or software using semantic versioning (e.g., x.y.z)
         rights: The license you select determines how your work can be used by others. [Read More](https://docs.scholarsphere.psu.edu/guides/licenses/)
       actor:
-        orcid: "This is a 16-digit identifier in the format ####-####-####-####"
+        orcid: "To link your ORCiD please visit [This link](https://guides.libraries.psu.edu/orcid)"
```

Comment:
For accessibility, it might be better to make this whole text a link and have it say something like "Click here to link your ORCiD" — though @srerickson might have other preferences.

Reply:
Don't have a strong preference - I'm okay with making the whole line of text a link.